sklearn_numba_dpex/common/tests/test_topk.py

```diff
@@ -0,0 +1,91 @@
+import dpctl.tensor as dpt
+import numpy as np
+import pytest
+
+from sklearn_numba_dpex.common.topk import topk, topk_idx
+from sklearn_numba_dpex.testing import _assert_array_equal_any
+from sklearn_numba_dpex.testing.config import float_dtype_params
+
+
+@pytest.mark.parametrize("dtype", float_dtype_params)
+@pytest.mark.parametrize("work_group_size", [4, 8, None])
+@pytest.mark.parametrize(
+    "array_in, expected_top_3, list_of_expected_top_3_idx",
```

Comment:
Is this more exact?
```suggestion
    "array_in, expected_top_3, list_of_possible_top_3_idx",
```

Also is there any specific tie breaking for this implementation? E.g. does it choose the smallest indices in case of tie break?

Reply:
Added comments for clarifying that there is no tie break strategy and that the output is not deterministic.